spec/slack-sup/commands/set_spec.rb

```diff
@@ -138,6 +138,30 @@
         )
       end
     end
+    context 'followup' do
+      it 'defaults to Thursday' do
+        expect(message: "#{SlackRubyBot.config.user} set followup").to respond_with_slack_message(
+          "Team S'Up followup day is on Thursday."
+        )
+      end
+      it 'shows current value of sup followup' do
+        team.update_attributes!(sup_followup_wday: 2)
+        expect(message: "#{SlackRubyBot.config.user} set followup").to respond_with_slack_message(
+          "Team S'Up followup day is on Tuesday."
+        )
+      end
+      it 'changes followup' do
+        expect(message: "#{SlackRubyBot.config.user} set followup friday").to respond_with_slack_message(
+          "Team S'Up followup day is now on Friday."
+        )
+        expect(team.reload.sup_followup_wday).to eq 5
+      end
+      it 'errors set on an invalid day' do
+        expect(message: "#{SlackRubyBot.config.user} set followup foobar").to respond_with_slack_message(
+          "Day _foobar_ is invalid, try _Monday_, _Tuesday_, etc. Team S'Up followup day is on Thursday."
+        )
+      end
+    end
```

Comment:
There're more specs below like for when you're not a team admin, etc. Please add.